lib/rules/time-min-milliseconds/index.js

```diff
@@ -101,34 +101,54 @@ function rule(minimum, options) {
 			}
 		}
 
+		/**
+		 * @param {string} propertyName
+		 * @returns {boolean}
+		 */
 		function isIgnoredProperty(propertyName) {
-			if (optionsMatches(options, 'ignore', 'delay') && DELAY_PROPERTIES.has(propertyName)) {
+			if (
+				optionsMatches(secondaryOptions, 'ignore', 'delay') &&
+				DELAY_PROPERTIES.has(propertyName)
+			) {
 				return true;
 			}
 
 			return false;
 		}
 
+		/**
+		 * @param {string} time
+		 * @returns {boolean}
+		 */
 		function isAcceptableTime(time) {
 			const parsedTime = valueParser.unit(time);
 
 			if (!parsedTime) return true;
 
-			if (parsedTime.number <= 0) {
+			const numTime = Number(parsedTime.number);
```

Comment:
[note] Avoiding implicit conversion from string to number (`parsedTime.number` is a string).